pom.xml

```diff
@@ -11,11 +11,7 @@
   <artifactId>negotiator</artifactId>
   <version>3.0.1</version>
   <modelVersion>4.0.0</modelVersion>
-  <packaging>pom</packaging>
-
-  <modules>
-    <module>negotiator-application</module>
-  </modules>
+  <packaging>war</packaging>
```

Comment:
it should be used jar as packaging, or we'd need to change the Docker ENTRYPOINT that is `ENTRYPOINT ["java","-jar", "-Dspring.profiles.active=docker", "/negotiator-application-3.0.1.jar"]`

Reply:
Should be fixed, please try